### PR TITLE
Adds two new party items with sounds.

### PR DESCRIPTION
**Pull request by razekpraxis**
Adds a party whistle with a 5 second cool down, as well as a one time use party popper that spawns confetti where the user is standing.

For fun!

### CODE REVIEW

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -910,17 +910,16 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 /datum/supply_packs/organic/party
 	name = "Party Equipment Crate"
 	contains = list(/obj/item/weapon/storage/box/drinkingglasses,
-					/obj/item/weapon/reagent_containers/food/drinks/shaker,
-					/obj/item/weapon/reagent_containers/food/drinks/bottle/patron,
-					/obj/item/weapon/reagent_containers/food/drinks/bottle/goldschlager,
-					/obj/item/weapon/reagent_containers/food/drinks/cans/ale,
-					/obj/item/weapon/reagent_containers/food/drinks/cans/ale,
```
Comment (Fox-McCloud) [inline code/datums/supplypacks.dm]: Why did you remove these. No. Put them back.

**Comment by razekpraxis** [comment]
you know, i don't know why i removed them. i must have left that in from testing stuff out
